docs/Basic-Guides.md

```diff
@@ -0,0 +1,142 @@
+# Basic Guides
+
+This guide shows you the basic steps you can try to playaround with ML-agents. 
+
+If you are not familiar with the [Unity Engine](https://unity3d.com/unity),
+we highly recommend the [Roll-a-ball tutorial](https://unity3d.com/learn/tutorials/s/roll-ball-tutorial) to learn all the basic concepts of Unity. 
+
+## Setting up ML-Agents within Unity
+
+1. Open Unity and use it to open folder `ml-agents/unity-environment`. 
+2. Go to `Edit` -> `Project Settings` -> `Player`
+3. For **each** of the platforms you target 
+(**`PC, Mac and Linux Standalone`**, **`iOS`** or **`Android`**):
+    1. Go into `Other Settings`.
+    2. Select `Scripting Runtime Version` to 
+    `Experimental (.NET 4.6 Equivalent)`
+    3. In `Scripting Defined Symbols`, add the flag `ENABLE_TENSORFLOW`. 
+    After typing in, press Enter.
+4. Go to `File` -> `Save Project`
+5. Restart the Unity Editor.
+
+![Project Settings](images/project-settings.png)
+
+## Setting up TensorflowSharp Support
+
+[Download](https://s3.amazonaws.com/unity-ml-agents/0.3/TFSharpPlugin.unitypackage) TensorflowSharp plugin, and import it into Unity once downloaded by double clicking on it.  You can see if it was successfully imported by checking the TensorFlow files in the Project window under `Assets` -> `ML-Agents` -> `Plugins` -> `Computer`. 
+
+**Note**: If you don't see anything under `Assets`, drag the `ml-agents/unity-environment/Assets/ML-Agents` folder under `Assets` within Project window.
+
+![Imported Tensorflowsharp](images/imported-tensorflowsharp.png)
+
+## Play an example environment using pretrained model
+
+1. In the Project window, go to `Assets` -> `ML-Agents` -> `Examples` -> `3DBall` folder and open the `3DBall` scene file. 
+2. In the Hierarchy window, click on `Ball3DAcademy` -> `Ball3DBrain`. 
+3. In the Inspector window, under `Brain (Script)` -> `Brain Type`, change the `Brain Type` to `Internal`. 
```

Comment:
3. On the **Ball3DBrain** object's **Brain** component, change the **Brain Type** to **Internal**